src/core/lib/iomgr/exec_ctx.h

```diff
@@ -97,7 +97,7 @@ class ExecCtx {
  public:
   /** Default Constructor */
 
-  ExecCtx() : flags_(GRPC_EXEC_CTX_FLAG_IS_FINISHED) {
```

Comment:
I didn't understand why this flag is removed. Could you please clarify

Reply:
Glad you asked... I think we actually want a conditional here - added a big block of text in a comment to explain.